source/shared/net/guides/deciding-wallet.rst

```diff
@@ -1,79 +1,100 @@
+
+Before you decide which wallet to use, it's important to know that you can’t import identities and accounts from the Mobile Wallet into the Desktop Wallet. Conversely, you can’t import identities and accounts from the Desktop Wallet into the Mobile Wallet. This is because they handle private keys in different ways. It’s therefore essential that you choose between the wallets before you start creating identities and accounts. You can, however, send GTU from one wallet to another.
 
 Which wallet is best for you?
 =============================
 
-If you want extra security in the form of a Ledger hardware device, want to create multi-signature transactions, and want to be able to create a baker account, the Desktop Wallet is the right choice.
-If you want to be able to access your accounts on the go, don’t want to run a node, don’t want to depend on a Ledger hardware device, and only need to create standard (single signature) transactions, the Mobile Wallet is the right choice.
+The wallets work on two different environments: the Desktop Wallet on a computer and the Mobile Wallet on a phone. The Mobile Wallet is more accessible and easier to use on a day-to-day basis whereas the Desktop Wallet has an extra layer of security in the form of a Ledger device.
 
-What are the requirements for each wallet?
-==========================================
+- **Desktop Wallet**: Choose this wallet if you want extra security in the form of a :ref:`Ledger<install-Ledger-app>` hardware device, want to create multi-signature transactions, and want to be able to create a baker account.
+
+- **Mobile Wallet**: Choose this wallet if you want to be able to access your accounts on the go, don’t want to run a node, don’t want to depend on a Ledger hardware device, and only need to create standard (single signature) transactions.
+
+The following provides more details on the differences between the wallets.
+
+How the wallets store your private keys
+=======================================
+
+One of the main differences between the two wallets lies in how they store your private keys.
+
+-  The Desktop Wallet stores your private keys on a Ledger that is secured by a PIN code and backed up by a recovery phrase. The private keys never leave the Ledger hardware device, and the Ledger doesn't require internet access, which means you can keep your private keys offline. However, this also means that you need the Ledger to sign the transactions you create in the Desktop Wallet, which makes it less convenient than the Mobile Wallet.
+
+- The Mobile Wallet creates and stores your private keys on the phone. This means it's easier to use because you don't need a Ledger device to make transactions. However, because your phone is connected to the internet, the wallet is more vulnerable to security breaches than the Desktop Wallet. Also, anyone with access to the mobile wallet can make transfers from the accounts in the mobile wallet.
 
```

Comment:
This is wrong: "Also, anyone with access to the mobile wallet can make transfers from the accounts in the mobile wallet."

The wallet is protected by passcode or fingerprint/faceid so only if the user hands out his passcode do other have that level of access. I think we should remove this sentence completely.